### PR TITLE
316531 Revamp ASP.NET MVC sample for embedding dasboard in iframe

### PR DESCRIPTION
**Pull request by anuabarnabsf4019**

### CODE REVIEW

File: IframeEmbedding/Controllers/HomeController.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+using System.Text;
+using System.Web.Mvc;
+
+namespace IframeEmbedding.Controllers
+{
+    public class HomeController : Controller
+    {
+        public ActionResult Index()
+        {
+            var nonce = Guid.NewGuid().ToString(); // random string
```
Comment (vinothkumar1732) [inline IframeEmbedding/Controllers/HomeController.cs]: Variable declaration is not good. Get the detail and knowledge from Nithya as she worked on Iframe SSO using token Embedding sample.

File: IframeEmbedding/Controllers/HomeController.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            var userInfo = EmbedProperties.UserEmail; // email address of the user
+            bool canSaveView = true; // enable or disable permission to create, open, update, delete view 
+            bool hasViews = true; // enable or disable the permission to check the views of the dashboard
+            bool hasExport = true; // enable or disable the permission to export the dashboards and widgets
+            bool hasDashboardComments = true; // enable or disable the permission to comment related actions to dashboards
+            bool hasWidgetComments = true; // enable or disable the permission to comment related actions to widgets
+            bool isMarkFavorite = true; // enable to disable the permission to make the dashboard favorite
+            double timeStamp = DateTimeToUnixTimeStamp(DateTime.UtcNow); // current time as UNIX time stamp
+            var expirationTime = "100"; // alive time of the token
```
Comment (vinothkumar1732) [inline IframeEmbedding/Controllers/HomeController.cs]: Do you know the value of 100 as expirationTime here?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,72 @@
...
+## Dashboard view
+
+![Dashboard View](https://github.com/boldbi/iframe-dashboard-asp-net-mvc-sample/assets/129487075/dd9a1ca5-7e92-4dec-aa0f-fd4b45b108ec)
+
+## Prerequisites
+
+ * [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)
+ * [.NET Framework 4.8](https://dotnet.microsoft.com/en-us/download/dotnet-framework)
+
+ #### Help link
```
Comment (vinothkumar1732) [inline README.md]: Remove this help link.

File: IframeEmbedding/IframeEmbedding.csproj.user [hunk 4]
```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (vinothkumar1732) [inline IframeEmbedding/IframeEmbedding.csproj.user]: This is unwanted and dynamically creating file. Remove this.

**Review by vinothkumar1732 — CHANGES_REQUESTED** [review]
Resolve the raised concerns.

File: README.md (hunk 3 above)
Comment (anuabarnabsf4019) [reply README.md]: removed

File: IframeEmbedding/Controllers/HomeController.cs (hunk 1 above)
Comment (anuabarnabsf4019) [reply IframeEmbedding/Controllers/HomeController.cs]: updated all variables

File: IframeEmbedding/Controllers/HomeController.cs (hunk 2 above)
Comment (anuabarnabsf4019) [reply IframeEmbedding/Controllers/HomeController.cs]: Expiration time checked and changed the value

File: IframeEmbedding/IframeEmbedding.csproj.user (hunk 4 above)
Comment (anuabarnabsf4019) [reply IframeEmbedding/IframeEmbedding.csproj.user]: Removed this file

**Review by vinothkumar1732 — APPROVED** [review]
Concerns were resolved and changes are fine.

**Pull request merged**